### PR TITLE
GUACAMOLE-1892: Fix the build issues related to librt for Ubuntu 22.04.

### PR DESCRIPTION
**Pull request by myjimmy**
I've fixed the build issues related to librt for Ubuntu 22.04.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -68,6 +68,10 @@ AC_CHECK_LIB([pthread], [pthread_create], [PTHREAD_LIBS=-lpthread
               AC_DEFINE([HAVE_LIBPTHREAD],,
                         [Whether libpthread was found])])
 
+# librt
+AC_CHECK_LIB([rt], [timer_create], [RT_LIBS=-lrt],
+             AC_MSG_ERROR("librt is required for performing timer-related operations"))
```
Comment (mike-jumper) [inline configure.ac:73]: It's not _universally_ required. This check should only be fatal where there is no `timer_create()` without linking in librt. If the test determines that `timer_create()` does not require linking with librt, this check should pass and `RT_LIBS` should be empty.

File: src/libguac/Makefile.am [hunk 2]
```diff
@@ -171,5 +171,6 @@ libguac_la_LDFLAGS =     \
     @UUID_LIBS@          \
     @VORBIS_LIBS@        \
     @WEBP_LIBS@          \
-    @WINSOCK_LIBS@
+    @WINSOCK_LIBS@       \
+    @RT_LIBS@
```
Comment (mike-jumper) [inline src/libguac/Makefile.am:175]: Please maintain alphabetical ordering.

**Review by mike-jumper — CHANGES_REQUESTED** [review]

File: src/libguac/Makefile.am (hunk 2 above)
Comment (myjimmy) [reply src/libguac/Makefile.am]: OK, I've changed code according to your opinion.

File: configure.ac [hunk 3]
```diff
@@ -68,6 +68,11 @@ AC_CHECK_LIB([pthread], [pthread_create], [PTHREAD_LIBS=-lpthread
               AC_DEFINE([HAVE_LIBPTHREAD],,
                         [Whether libpthread was found])])
 
+# librt
+RT_LIBS=
+AC_CHECK_LIB([rt], [timer_create], [RT_LIBS=-lrt],
+             AC_MSG_ERROR("librt is required for performing timer-related operations"))
```
Comment (mike-jumper) [inline configure.ac:74]: Please see https://github.com/apache/guacamole-server/pull/476#discussion_r1447776140:

> It's not _universally_ required. This check should only be fatal where there is no `timer_create()` without linking in librt. If the test determines that `timer_create()` does not require linking with librt, this check should pass and `RT_LIBS` should be empty.

As written, this test will fail the build if librt is absent, even if the platform in question doesn't require librt for `timer_create()`.

**Review by mike-jumper — CHANGES_REQUESTED** [review]

File: configure.ac (hunk 3 above)
Comment (myjimmy) [reply configure.ac]: @mike-jumper I've fixed the above code specified by you.

Comment (myjimmy) [reply configure.ac]: @mike-jumper I've fixed the above code specified by you.

**Review by mike-jumper — APPROVED** [review]

**Pull request merged**